lib/measured/base.rb

```diff
@@ -27,6 +36,10 @@ def method_missing(method, *args)
 require "measured/arithmetic"
 require "measured/unit"
 require "measured/conversion"
+require "measured/conversion_builder"
 require "measured/conversion_table"
 require "measured/measurable"
-require "measured/case_sensitive_measurable"
+  
+module Measured
+  private_constant :Measurable
```

Comment:
Easiest way to prevent inheriting from `Measurable` is to make it a hidden detail of the module. Are there better options, @kmcphillips ?